src/main/java/donnafin/ui/ClientPanel.java

```diff
@@ -141,6 +133,7 @@ private void changeTab(Node node) {
             refresh();
             attributeDisplayContainer.getChildren().add(node);
         });
-        attributeDisplayContainer.setVgrow(node, Priority.ALWAYS);
+        VBox.setVgrow(node, Priority.ALWAYS);
```

Comment:
How does this work. Arent we specifyign attributeDisplayContainer to be the one that grows

Reply:
nope, we are not, this line actually sets node to be always growing. [`setVgrow`](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/layout/VBox.html#setVgrow-javafx.scene.Node-javafx.scene.layout.Priority-) is a static method, it never relied on `attributeDisplayContainer`

Reply:
aight. Understand. Went to test it out as well